tests/cli/test_core_commands.py

```diff
@@ -29,6 +29,7 @@ def mock_subprocess_run(mocker: MockerFixture) -> MagicMock:
 @pytest.mark.parametrize("port", [8081, None])
 @pytest.mark.parametrize("reload", [True, False, None])
 @pytest.mark.parametrize("web_concurrency", [2, None])
+@pytest.mark.parametrize("app_dir", [Path.cwd(), None])
```

Comment:
Should probably test for something that's not the default value and the application file be placed under that path.